<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOrderWindow</class>
 <widget class="QMainWindow" name="AddOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить заказ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="table_label">
      <property name="text">
       <string>Столик:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="table_comboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="dishes_label">
      <property name="text">
       <string>Блюда:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="dishes_tableWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>Блюдо</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_order_pushButton">
      <property name="text">
       <string>Сохранить заказ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>